osquery/sql/tests/sql_tests.cpp

```diff
@@ -154,4 +154,11 @@ TEST_F(SQLTests, test_sql_sha256) {
   EXPECT_EQ(d[0]["test"],
             "9f86d081884c7d659a2feaa0c55ad015a3bf4f1b2b0b822cd15d6c15b0f00a08");
 }
+
+TEST_F(SQLTests, test_sql_regex_replace) {
```

Comment:
Not sure how is their test coverage but I would really like to see tests for empty _input_ and _find_string_, one with no match, one on a NULL input and one from a table.